gdraw/gtabset.c

```diff
@@ -175,8 +177,15 @@ static int DrawTab(GWindow pixmap, GTabSet *gts, int i, int x, int y ) {
         GDrawFillRect(pixmap,&r,gts->g.box->active_border);
     }
 
-    GDrawDrawText(pixmap,x+(gts->tabs[i].width-gts->tabs[i].tw)/2,y+gts->rowh-gts->ds,
-	    gts->tabs[i].name,-1,fg);
+    int nx = x+(gts->tabs[i].width-gts->tabs[i].tw)/2;
+    int ny = y+gts->rowh-gts->ds;
+    int nx1 = GDrawDrawText(pixmap,nx,ny,gts->tabs[i].name,-1,fg);
+    if (gts->closable) {
+        nx1 += (GTS_TABPADDING/2-5);
+        Color xcol = GResourceFindColor("GTabSet.CloseColor",0xff0000);
+        GDrawDrawLine(pixmap,nx+nx1,ny,nx+nx1+10,ny-10,xcol);
```

Comment:
I'm a bit wary about hard-coding another of these rather than providing an image option, perhaps with this as a backup.

Reply:
The images all look terrible in HiDPI. 

It's not a problem, IMO, to add another one of these until we solve that problem in general. That's another issue entirely.